lib/timex.ex

```diff
@@ -1375,15 +1375,16 @@ defmodule Timex do
   Same as `shift(date, Duration.from_minutes(5), :duration)`.
   """
   @spec add(Convertable.t, Duration.t) ::
-    Date.t | NaiveDateTime.t | DateTime.t | {:error, term}
+  Types.valid_datetime | | AmbiguousDateTime.t | {:error, term}
```

Comment:
You have a typo here (two `|`), which is breaking the build. Could you fix that up?